app/Services/Payment/PaymentService.php

```diff
@@ -42,4 +42,11 @@ public function manualPayment($invoice) :?Payment
 
         return $payment;
     }
+    
+    public function sendEmail($contact = null)
+    {
+        $send_email = new SendEmail($this->payment);
+
+        return $send_email->sendEmail(null, $contact);
```

Comment:
We have moved from __invokable to plain classes, we will use a default method name of ->run() for all service methods